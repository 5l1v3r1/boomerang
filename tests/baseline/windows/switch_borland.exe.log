COMPUTED JUMP at 0x0x40115b, pDest = m[(r24 * 4) + 0x401162]
COMPUTED JUMP at 0x0x403bb9, pDest = m[(r25 * 4) + 0x403bc0]
ERROR: attempted to decode outside any known segment 0x0x100403a20
ERROR: attempted to decode outside any known segment 0x0x100403a68
finding final parameters for proc4
finding final parameters for proc3
after removing matching assigns (%pc).
Could not find number of cases for n-way at address 0x0x403bb9
=== about to restart decompilation of proc2 because indirect jumps or calls have been analysed

Error: Instruction at 0x0x403c58 branches beyond end of section, to 0x0x100403b8b
Error: Instruction at 0x0x403c61 branches beyond end of section, to 0x0x100403b8b
Error: Instruction at 0x0x403c37 branches beyond end of section, to 0x0x100403b8b
Error: Instruction at 0x0x403ce2 branches beyond end of section, to 0x0x100403b8b
Error: Instruction at 0x0x403ccd branches beyond end of section, to 0x0x100403b8b
after removing matching assigns (%pc).
### WARNING: iteration limit exceeded for dfaTypeAnalysis of procedure proc2 ###
in proc proc2 adding addrExp r28{0} - 4 to local table
in proc proc2 adding addrExp r28{0} - 0x528 to local table
in proc proc2 adding addrExp r28{0} - 0x52c to local table
in proc proc2 adding addrExp r28{0} - 0x530 to local table
in proc proc2 adding addrExp r28{0} - 0x4c4 to local table
in proc proc2 adding addrExp r28{0} - 0x4c8 to local table
in proc proc2 adding addrExp r28{0} - 0x4d4 to local table
in proc proc2 adding addrExp r28{0} - 0x4d0 to local table
in proc proc2 adding addrExp r28{0} - 0x4cc to local table
in proc proc2 adding addrExp r28{0} - 0x4c0 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 to local table
finding final parameters for proc2
potential param r28
found new parameter r28
potential param m[r28{0} + 16]
found new parameter m[r28{0} + 16]
finding final parameters for proc2
potential param r28
found new parameter r28
potential param m[r28{0} + 16]
found new parameter m[r28{0} + 16]
after removing matching assigns (t[WILDSTR]).
after removing matching assigns (%pc).
in proc proc1 adding addrExp r28{0} - 4 to local table
in proc proc1 adding addrExp r28{0} - 8 to local table
in proc proc1 adding addrExp r28{0} - 12 to local table
in proc proc1 adding addrExp r28{0} - 16 to local table
in proc proc1 adding addrExp r28{0} - 20 to local table
in proc proc1 adding addrExp r28{0} - 24 to local table
in proc proc1 adding addrExp r28{0} - 28 to local table
in proc proc1 adding addrExp r28{0} - 32 to local table
finding final parameters for proc1
potential param r28
found new parameter r28
potential param m[r28{0} + 4]
found new parameter m[r28{0} + 4]
after removing matching assigns (t[WILDSTR]).
after removing matching assigns (%pc).
=== about to restart decompilation of main because indirect jumps or calls have been analysed

after removing matching assigns (t[WILDSTR]).
after removing matching assigns (%pc).
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 4 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
could not set type for unknown parameter expression m[r28{0} + 4]
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 8 to local table
### WARNING: iteration limit exceeded for dfaTypeAnalysis of procedure proc2 ###
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 to local table
in proc proc2 adding addrExp r28 - 4 to local table
in proc proc2 adding addrExp r28 to local table
finding final parameters for proc1
potential param r28
found new parameter r28
potential param m[r28{0} + 4]
found new parameter m[r28{0} + 4]
