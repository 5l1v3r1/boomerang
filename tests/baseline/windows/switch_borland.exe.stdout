Boomerang alpha 0.3.2 19/Aug/2010
loading...
decoding entry point...
decoding anything undecoded...
finishing decode...
found 5 procs
decompiling...
 considering main
  considering proc1
   considering proc2
    considering proc4
    decompiling proc4
    considering proc3
    decompiling proc3
   decompiling proc2
   reconsidering proc2
    considering proc8
    decompiling proc8
    considering proc5
     considering proc12
     decompiling proc12
     considering proc11
     decompiling proc11
    decompiling proc5
    considering proc6
    decompiling proc6
    considering proc7
    decompiling proc7
    considering proc9
    decompiling proc9
    considering proc10
    decompiling proc10
   decompiling proc2
  decompiling proc1
 decompiling main
 reconsidering main
 decompiling main
global type analysis for main
global type analysis for proc1
global type analysis for proc2
global type analysis for proc3
global type analysis for proc4
global type analysis for proc5
global type analysis for proc6
global type analysis for proc7
global type analysis for proc8
global type analysis for proc9
global type analysis for proc10
global type analysis for proc11
global type analysis for proc12
 transforming out of SSA form proc2 with 193 BBsiiiiiiiiiiiiiiiiiiiiiiii